includes/class-img-converter.php

```
@@ -574,60 +575,48 @@ public static function add_img_into_queue( $img_path, $type = 'webp' ) {
-				'shutdown',
-				function () {
-					if ( null !== self::$img_info_cache ) {
-						update_option( 'wppo_img_info', self::$img_info_cache );
-					}
-				}
-			);
-			self::$shutdown_hook_registered = true;
-		}
+	private static function update_img_info_atomic( callable $callback ): void {
+		$img_info = get_option( 'wppo_img_info', array() );
+		$new_info = $callback( $img_info );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$new_info'. (undefined)

(UnusedLocalVariable)